oscaralanpierce/skyrim_inventory_management_frontend
# Add placeholder login page

## Context

[**Build deployable app scaffold using new stack**](https://trello.com/c/yFori2Hb/228-build-deployable-app-scaffold-using-new-stack)

The attached card involves adding three new pages to the stock Vite app previously created: a homepage, a placeholder login page (no login functionality yet) and a stock 404 page. The homepage is added in #11. This PR adds the login page and updates the homepage to link to it.

## Changes

* Create placeholder `LoginPage` component with tests and stories
* Add route for login page so it can be used within the site
* Add link to the login page from the homepage
* Update tests and stories to accommodate links
* Add shitty favicon (will need to update in the future but didn't want to spend time on this card)
* Remove stock images included by Vite

## Considerations

We'd like to begin supporting screen sizes as small as 320 px (standard mobile). The homepage here still looks weird below 425. So for this PR, I'm treating 425px as the minimum supported screen width and will update in the future.

## Manual Test Cases

Run the app in the development environment. See that the homepage now has a link to the login page. Clicking the link should bring you to the login page. Clicking the link on the login page should take you back to the homepage.

## Screenshots and GIFs

### 425px

#### Homepage

<img width="424" alt="HomePage-425" src="https://user-images.githubusercontent.com/5115928/220205291-5351a84c-d496-402a-a26d-a3c9e41d50fd.png">

#### Login Page

<img width="423" alt="LoginPage-425" src="https://user-images.githubusercontent.com/5115928/220205323-7602515c-6298-40b8-8546-8e6884994051.png">

### 601px

#### Homepage

<img width="603" alt="HomePage-601" src="https://user-images.githubusercontent.com/5115928/220205351-1bf03923-13f0-4468-9412-26354c8e6a1b.png">

#### Login Page

<img width="599" alt="LoginPage-601" src="https://user-images.githubusercontent.com/5115928/220205388-b6aef0ff-137d-4d9e-bc90-6983248ef9ad.png">

### 769px

#### Homepage

<img width="770" alt="HomePage-769" src="https://user-images.githubusercontent.com/5115928/220205412-7ff163b2-376d-4f1d-bc0d-747bf49877e2.png">

#### Login Page

<img width="767" alt="LoginPage-769" src="https://user-images.githubusercontent.com/5115928/220205434-f60492ea-4d36-4735-a89b-b5adaf68b510.png">

### 1025px

#### Homepage

<img width="1025" alt="HomePage-1025" src="https://user-images.githubusercontent.com/5115928/220205468-463bc8c6-c1ec-445f-badc-4b8f68837946.png">

#### Login Page

<img width="1025" alt="LoginPage-1025" src="https://user-images.githubusercontent.com/5115928/220205492-faf73186-c1f4-4b01-97a9-940dc0a42b60.png">

### 1201px

#### Homepage

<img width="1200" alt="HomePage-1201" src="https://user-images.githubusercontent.com/5115928/220205535-ae7b1248-88b6-4e1e-8631-0de37384b42c.png">

#### Login Page

<img width="1200" alt="LoginPage-1201" src="https://user-images.githubusercontent.com/5115928/220205556-4de8fdaf-04ce-4d42-a4f2-e8ebd9e20f36.png">

### Large Desktop

#### Homepage

<img width="1918" alt="HomePage-Full" src="https://user-images.githubusercontent.com/5115928/220205590-049a5b59-b810-416d-8f43-e698aaaea896.png">

#### Login Page

<img width="1919" alt="LoginPage-Full" src="https://user-images.githubusercontent.com/5115928/220205606-0a25872d-f85f-4d01-b6b3-2401d92cd2b2.png">
